resources/ts/Pages/Integrations/New.tsx

```diff
@@ -33,41 +33,46 @@ const integrationTypes = (t: TFunction) => [
 
 const pricing = (t: TFunction, subscriptions: Subscription[]) => {
   const getInfoForType = (type: string) => {
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     const data = subscriptions.find(
       (sub) => sub.category.toLowerCase() === type
-    )!;
+    );
 
     return {
-      price: data.price,
-      currency: data.currency,
+      price: data?.price ?? "Te bepalen",
+      currency: data?.currency ?? "EUR",
```

Comment:
Maybe add a todo comment.

`// TODO: Remove defaults once all subscriptions have been added in the backend`